sympy/printing/latex.py

```diff
@@ -415,12 +416,9 @@ def convert(expr):
                     _tex += term_tex
                     last_term_tex = term_tex
                 return _tex
+        one_by_one =  Pow(1, -1, evaluate=False)
 
-        if denom is S.One:
-            # use the original expression here, since fraction() may have
-            # altered it when producing numer and denom
-            tex += convert(expr)
```

Comment:
Is it possible to preserve this line if `Pow(1, -1, evaluate=False) not in expr.args` is added to the above condition?`